static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -9,17 +9,26 @@ Once initialized in a project, DVC populates its installation directory
 - `.dvc/config.local` - this is a local configuration file, that will overwrite
   options in `.dvc/config`. This is useful when you need to specify private
   options in your config, that you don't want to track and share through git.
-  The local config file can be edited by hand or with a special command: `dvc
-  config --local`.
-
-- `.dvc/cache` - the cache directory will contain your data files. (The data
+  The local config file can be edited by hand or with a special command:
+  `dvc config --local`.
+
+- `.dvc/cache` - the cache directory will contain your data files. The data file
```

Comment:
let's make a separate section in this file, that describes this structure. It will be too long for a list item.

Reply:
@shcheklein  Something like this?
![Screenshot from 2019-05-22 09-36-18](https://user-images.githubusercontent.com/35191225/58146755-182b0080-7c75-11e9-82d0-85040cb6ad01.png)